### PR TITLE
foreman-maintain replaced to satellite-maintain

### PR DESCRIPTION
**Pull request by tstrych**
we should use downstream name for foreman-maintain and that is satellite maintain

I did rename files from `/robottello/cli/fm*` to `/robottelo/cli/sm*` and I didn't found any usages of these files in robotello, but it didn't feel quite right so maybe I just didn't found the usages or I need explanation why we use them. 

other usages of FM are in: `upgrade.yaml`  and `subscription.yaml`

### CODE REVIEW

**Review by swadeley — APPROVED** [review]
ACK

File: tests/foreman/sys/test_katello_certs_check.py [hunk 1]
```diff
@@ -131,7 +131,7 @@ def validate_output(self, result, cert_data):
         # validate correct installer used
         pattern, count = 'satellite-installer --scenario satellite', 2
         assert len(re.findall(pattern, result.stdout)) == count
-        pattern, count = 'foreman-installer --scenario katello', 0
```
Comment (Gauravtalreja1) [inline tests/foreman/sys/test_katello_certs_check.py:134]: I believe this should not be changed, as this step/test validates if the correct installer is used, @omkarkhatavkar can confirm?

**Review by Gauravtalreja1 — CHANGES_REQUESTED** [review]

File: tests/foreman/sys/test_katello_certs_check.py (hunk 1 above)
Comment (tstrych) [reply tests/foreman/sys/test_katello_certs_check.py:134]: it tests that there is 0 `katello` options, in installer, it doesn't matter which installer you are looking at

Comment (omkarkhatavkar) [reply tests/foreman/sys/test_katello_certs_check.py:134]: @Gauravtalreja1 is correct, earlier we used to get a lot of issues with the cert generation command we wanted to ensure that foreman-installer options should not be seen. Please revert this.

**Comment by Gauravtalreja1** [comment]
@tstrych any status update?

File: tests/foreman/sys/test_katello_certs_check.py [hunk 2]
```diff
@@ -129,7 +129,7 @@ def validate_output(self, result, cert_data):
         assert cert_data['success_message'] in result.stdout
 
         # validate correct installer used
-        pattern, count = 'satellite-installer --scenario satellite', 2
+        pattern, count = 'foreman-installer --scenario satellite', 2
```
Comment (Gauravtalreja1) [inline tests/foreman/sys/test_katello_certs_check.py:132]: I guess, this should not be changed for this test, as it checks first if the right installer is used, and the upstream installer is not used, @omkarkhatavkar can confirm?

Comment (omkarkhatavkar) [reply tests/foreman/sys/test_katello_certs_check.py:132]: It should be `satellite-installer`

**Comment by omkarkhatavkar** [comment]
@tstrych  Can you please run the PRT and see if there are any failures related to this change?

File: tests/foreman/sys/test_katello_certs_check.py (hunk 2 above)
Comment (tstrych) [reply tests/foreman/sys/test_katello_certs_check.py:132]: good catch thanks :)

**Comment by tstrych** [comment]
@omkarkhatavkar sure, I can run it, but I don't see how it will provide much valid information. This should not change the logic, so if the test fails, it should fail even before.

**Comment by tstrych** [comment]
trigger: test-robottelo
pytest: tests/foreman/api/test_remoteexecution.py tests/foreman/cli/test_puppetplugin.py tests/foreman/cli/test_rhcloud_inventory.py tests/foreman/installer/test_installer.py tests/foreman/longrun/test_remote_DB.py tests/foreman/sys/test_foreman_service.py tests/foreman/sys/test_katello_certs_check.py tests/foreman/ui/test_contenthost.py tests/foreman/ui/test_ldap_authentication.py

**Review by mshriver — APPROVED** [review]

**Pull request merged**